CHANGELOG.md

```diff
@@ -28,7 +29,7 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 
 ### Contributors
 
-- None
+- Zach Angell
```

Comment:
Totally up to you @zangell44 but if you want to include a link to yourself, feel free to (using markdown links) (e.g., `[Zach Angell](https://github.com/zangell44)` or your personal homepage, etc.).  I'll hold off on merging for a bit in case you want to do this.

Reply:
Much appreciated! A little visibility doesn't hurt. I just pushed a change that links to my GH